docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -0,0 +1,137 @@
+most of the wolves were photographed in the snow and therefore had a white background while all the dogs were
+photographed in the grass and therefore had a green background.
+
+The check is based on calculating the predictive power score (PPS) of each image
+property. For more details you can read here `how the PPS is calculated
+<#how-is-the-predictive-power-score-pps-calculated>`__.
+
+What is a problematic result?
+-----------------------------
+
+Image properties with a high predictive score can indicate that there is a bias in the dataset, as a single property
+can predict the label successfully using simple classic ML algorithms.
+
+This means that a deep learning algorithm may accidentally learn these properties instead of more accurate complex
+abstractions. For example, in the dataset of wolves and dogs photographs, the brightness of the image may be used to
+predict the label "wolf" easily.
+
+
+How do we calculate for different vision tasks?
+-----------------------------------------------
+
+* For classification tasks, this check uses PPS to predict the class by image properties.
+* For object detection tasks, this check uses PPS to predict the class of each
+  bounding box, by the image properties of that specific bounding box.
+  This means that for each image, this check crops all the sub-images defined by bounding
+  boxes, and uses them as inputs as though they were regular classification dataset images.
+
+How is the Predictive Power Score (PPS) calculated?
+---------------------------------------------------
+The properties' predictive score results in a numeric score between 0 (feature has
+no predictive power) and 1 (feature can fully predict the label alone).
+
+The process of calculating the PPS is the following:
+
+The properties' predictive score results in a numeric score between 0 (feature has no predictive power) and 1 (feature
+can fully predict the label alone).
+
+The process of calculating the PPS is the following:
+"""
+#%%
+# 1. Extract from the data only the label and the feature being tested
+# 2. Drop samples with missing values
+# 3. Keep 5000 (this is configurable parameter) samples from the data
+# 4. Preprocess categorical columns. For the label using `sklearn.LabelEncoder` and for the feature
+#    using `sklearn.OneHotEncoder`
+# 5. Partition the data with 4-fold cross-validation
+# 6. Train decision tree
+# 7. Compare the trained model's performance with naive model's performance as follows:
+#
+# Regression: The naive model always predicts the median of the label column,
+# the metric being used is MAE and the PPS calculation is: :math:`1 - \frac{\text{MAE model}}{\text{MAE naive}}`
+#
+# Classification: The naive model always predicts the most common class of
+# the label column, The metric being used is F1 and the PPS calculation is:
+# :math:`\frac{\text{F1 model} - \text{F1 naive}}{1 - \text{F1 naive}}`
```

Comment:
in the pdf - the latex for these wasn't parsed properly (the plaintext is shown)
some syntax error?

Reply:
@shir22 it's a bug in the make docs/ boilerplate that's supposed to OK in the actual documentation, it looks the same in the PropertyLabelCorrelationChange in the make docs but OK at our website